gdkx11/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![allow(clippy::type_complexity)]
 
 pub use ffi;
+pub use x11;
```

Comment:
No gdk and glib here? Also this has no prelude?

Reply:
Ah it seems like I missed them on gtk4-rs as  well ^^ I have added them & removed the pango/gdk-pixbuf dependencies from gdkx11 because it doesn't need them :)